modules/vector-sets/vset.c

```diff
@@ -106,6 +106,7 @@
 #define _POSIX_C_SOURCE 200809L
 
 #include "../../src/redismodule.h"
+#include "../../src/sds.h"
```

Comment:
The vectorset module is independent, and we should not introduce core code into it.

Reply:
Would `RedisModuleString` be the appropriate choice for string manipulation in the module?

Reply:
Replaces sds with `RedisModuleString` in 50e52ff703c33d19720496dc4259ced95dce8666